system/jlib/jsmartsock.cpp

```diff
@@ -449,6 +480,15 @@ StringBuffer & CSmartSocketFactory::getUrlStr(StringBuffer &url, bool useHostNam
     return url;
 }
 
-ISmartSocketFactory *createSmartSocketFactory(const char *_socklist, bool _retry, unsigned _retryInterval, unsigned _dnsInterval) {
+ISmartSocketFactory *createSmartSocketFactory(IPropertyTree &service, const char *defPort, bool _retry, unsigned _retryInterval, unsigned _dnsInterval)
+{
+    const char *name = service.queryProp("@name");
+    DBGLOG("creating smart socket service client: %s, %s", service.getPropBool("@tls") ? "manual TLS" : "TCP", name ? name : "unnamed");
+    return new CSmartSocketFactory(service, defPort, _retry, _retryInterval, _dnsInterval);
+}
+
+ISmartSocketFactory *createSmartSocketFactory(const char *_socklist, bool _retry, unsigned _retryInterval, unsigned _dnsInterval)
+{
+    DBGLOG("creating smart socket: %s", _socklist);
```

Comment:
Could this tracing become excessive?  Same for the function above.  Possibly not since it is a factory